.github/workflows/e2e-test-no-domain.yaml

```diff
@@ -18,10 +18,11 @@ jobs:
     if: ${{ github.event.label.name == 'e2e' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' }}
     name: E2E Testing with no domain
     runs-on: ubuntu-latest
+    permissions:
```

Comment:
See: https://github.com/aws-actions/configure-aws-credentials#usage

Reply:
To save the hop 
> # These permissions are needed to interact with GitHub's OIDC Token endpoint.